### PR TITLE
Setting up Travis auto deployment for Frontend

### PR DESCRIPTION
**Pull request by UladBohdan**

### CODE REVIEW

**Comment by UladBohdan** [comment]
[to find a way to deliver built code to the server instead of building it there.](https://trello.com/c/Q6ropNjg/210-to-find-a-way-to-deliver-built-code-to-the-server-instead-of-building-it-there)

File: deploy-dev.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#!/usr/bin bash
+set -e
```
Comment (Drapegnik) [inline deploy-dev.sh]: please move this file to `bin/` folder

File: package.json [hunk 2]
```diff
@@ -5,6 +5,7 @@
   "scripts": {
     "start": "npm run dev",
     "start:prod": "npm run build && cross-env NODE_ENV=production node server.js",
+    "start:prod:remote": "node server.js",
```
Comment (Drapegnik) [inline package.json:8]:
```suggestion
```

File: pm2-config.json [hunk 3]
```diff
@@ -6,14 +6,14 @@
       "WIR_ENV": "staging",
       "NODE_ENV": "production",
     },
-    "args": "run start:prod -- --backend_url=http://localhost:8080 --port=3000"
+    "args": "run start:prod:remote -- --backend_url=http://localhost:8080 --port=3000"
```
Comment (Drapegnik) [inline pm2-config.json:9]: `"script": "node"`, and:
```suggestion
    "args": "server.js --backend_url=http://localhost:8080 --port=3000"
```

**Review by Drapegnik — DISMISSED** [review]

File: pm2-config.json (hunk 3 above)
Comment (UladBohdan) [reply pm2-config.json:9]: I don't like the suggestion.
You're hardcoding the script into the config file.
Furthermore - it is repeated in this file.

File: deploy-dev.sh (hunk 1 above)
Comment (UladBohdan) [reply deploy-dev.sh]: Done.

**Review by Drapegnik — DISMISSED** [review]

**Pull request merged**